e2e/nx-run/src/run.test.ts

```diff
@@ -198,6 +198,15 @@ describe('Nx Running Tests', () => {
           scripts: {
             [target]: `echo ${expectedOutput}`,
           },
+          nx: {
+            targets: {
+              [target]: {
+                configurations: {
+                  production: {},
+                },
+              },
+            },
+          },
```

Comment:
Were you going to do something with this?

Reply:
Its required since we don't infer the production configuration from .env.production existing anymore

Reply:
We have to explicitly list that configuration if we want to be able to read env variables from the file / run with --configuration production

Reply:
I was, yes. It looks like the e2e I had added somehow got lost 🤔